gto-support-db/src/main/java/org/ccci/gto/android/common/db/Expression.java

```diff
@@ -760,4 +800,89 @@ public Unary createFromParcel(@NonNull final Parcel in, @Nullable final ClassLoa
             }
         }
     }
+
+    public static class Aggregate extends Expression {
+        static final String COUNT = "COUNT";
+        static final String MAX = "MAX";
+        static final String MIN = "MIN";
+        static final String SUM = "SUM";
+
+        @NonNull
+        private final String mOp;
+        @NonNull
+        private final Expression mExpr;
```

Comment:
Hmm, I think for now let's narrow the scope for our current use cases. We can figure out subselects in the future if the need arises, but expressions may not be correct for those, and we may want to keep subselects separate from aggregate functions